src/coreComponents/constitutive/docs/ConstantPermeability.rst

```diff
@@ -0,0 +1,33 @@
+.. _KozenyCarmanPermeability:
+
+
+####################################################
+Constant Permeability Model
+####################################################
+
+
+Overview
+======================
+
+This model is used to define a diagonal permeability tensor that does not depend on any primary variable.
+
+Parameters
+======================
+
+
+The following attributes are supported:
+
+.. include:: /coreComponents/schema/docs/CarmanKozenyPermeability.rst
```

Comment:
This seems to be documenting Kozeny-Carman, which is not a constant poro model

Reply:
wrong link